spaceship/lib/spaceship/connect_api/models/app.rb

```
@@ -0,0 +1,508 @@
+      attr_accessor :primary_locale
+      attr_accessor :is_opted_in_to_distribute_ios_app_on_mac_app_store
+      attr_accessor :removed
+      attr_accessor :is_aag
+      attr_accessor :available_in_new_territories
+      attr_accessor :content_rights_declaration
+      attr_accessor :app_store_versions
+      attr_accessor :prices
+
+      # Only available with Apple ID auth
+      attr_accessor :distribution_type
+      attr_accessor :educationDiscountType
```

RuboCop (1.89.0)

[convention] Group together all `attr_accessor` attributes.

(Style/AccessorGrouping)